### DISCUSSION TITLE
番号付きリストを `1.` 以外で開始した場合でも、“1.” から開始してしまう

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#461] [要望・アイデア・不具合]
CommonMark では、`1.` 以外で開始した番号付きリストはその番号から開始する仕様になっています。

>> If the list item is ordered, then it is also assigned a start number, based on the ordered list marker.
>
> https://spec.commonmark.org/0.30/#:~:text=its%20list%20marker.-,If%20the%20list%20item%20is%20ordered%2C%20then%20it%20is%20also%20assigned%20a%20start%20number%2C%20based%20on%20the%20ordered%20list%20marker.,-Exceptions%3A

Qiita では CommonMark に準拠しているはずですが、なぜか `1.` 以外で開始した番号付きリストであっても “1.” から始まってしまいます 😨 

![image](https://user-images.githubusercontent.com/75878492/231071539-af9e9f75-565a-47e4-a270-f872fd1835ea.png)

<details><summary>サンプルコード</summary>
<p>

```markdown
text text text

10. list item
10. list item
10. list item
``` 

text text text

10. list item
10. list item
10. list item

</p>
</details> 

CommonMark のように、開始したリストマーカーの数字から番号付きリストを開始してほしいです。

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
本当ですね…💦どこかのタイミングで修正いたします！🙏
教えていただきありがとうございます！
